Vladyslav_Gurevych/HW5/Task5**.py

```diff
@@ -0,0 +1,16 @@
+import re
+
+file = open("index.html")
+html = file.read()
+file.close()
+result = []
+id = re.findall("\"[a-z]{5,}\"", html)
```

Comment:
![image](https://user-images.githubusercontent.com/25909810/188006456-39ddee54-23d0-4725-a987-d422609ba7ae.png)
Try to avoid such naming. Use item_id or something like this instead